packages/components/text/src/text.styles.ts

```diff
@@ -48,6 +48,8 @@ const getTone = (tone: string) => {
       return `color: ${designTokens.colorSurface};`;
     case 'warning':
       return `color: ${designTokens.colorWarning};`;
+    case 'critical':
+      return `color: ${designTokens.colorError40};`;
```

Comment:
If we support this new tone, it should be added to the list of options in `tone` prop as well.